Assets/MRTK/SDK/Experimental/ColorPicker/ColorPicker.cs

```diff
@@ -0,0 +1,352 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using UnityEngine;
+using TMPro;
+using Microsoft.MixedReality.Toolkit.UI;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.ColorPicker
```

Comment:
Please add an [Experimental] attribute to this component to alert people that it is exerimental. See [Experimental Guidelines](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/ExperimentalFeatures.html) for more.

Reply:
added